notebooks/ivf_flat_example.ipynb

```diff
@@ -0,0 +1,803 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def calc_recall(ann_idx, true_nn_idx):
</pre><p>We should consider pulling these out into a python file (maybe <code>common.py</code>?) in the <code>notebooks/</code> directory so we can share the code with other notebooks. It'll also clean up the notebook a little bit since it's not a particularly relevant detail for IVF-flat demonstration. It would also be worth quickly updating the ivf-pq notebook to use at least the <code>calc_recall</code> function from this notebook.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>moved to utils.py</p>